lib/Dependency.js

```diff
@@ -7,7 +7,7 @@ function Dependency() {
 }
 module.exports = Dependency;
 
-Dependency.prototype.isEqualResource = function( /* other */ ) {
```

Comment:
Understandable. The problem was that eslint and js-beautify don't agree with eachother about if there should be a space before and after the comment. js-beautify wants it with spaces, eslint without.

An alternative way of fixing this would be to add a comment add the top, `/* eslint-disable keyword-spacing */` (temporarily). After js-beautify is removed we can remove that line.